packages/ui/src/primitives/tailwind/Tooltip/index.tsx

```diff
@@ -28,7 +28,7 @@ import { twMerge } from 'tailwind-merge'
 
 interface TooltipProps {
   title: string
-  direction?: 'top' | 'bottom' | 'left' | 'right'
+  direction?: 'top' | 'bottom' | 'left' | 'right' | 'bottomRight'
```

Comment:
this seems to be a one-off case
we can provide the `className` prop to Tooltip in which we can add the absolute positions